tests/Tests.FeatureManagement/Pages/RazorTestAll.cshtml.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.FeatureManagement.Mvc;
+
+namespace Tests.FeatureManagement.Pages
+{
+    [FeatureGate(Features.ConditionalFeature, Features.ConditionalFeature2)]
+    public class RazorTestAllModel : PageModel
+    {
+        public IActionResult OnGet()
+        {
+            return StatusCode(200);
```

Comment:
> StatusCode(200)

Should we use `HttpStatusCode.OK` instead? [HttpStatusCode](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?view=netcore-3.1)

Reply:
Updated to `OkResult`. Now status code is avoided completely.